ui/compose.lua

```diff
@@ -40,10 +38,7 @@ minetest.register_on_player_receive_fields(function(player, formname, fields)
 
 	local name = player:get_player_name()
     if fields.send then
-        local id = mail.new_uuid()
-        if mail.selected_idxs.message[name] then
-            id = mail.selected_idxs.message[name]
-        end
+        local id = mail.selected_idxs.message[name] or mail.new_uuid()
```

Comment:
```suggestion
	local id = mail.selected_idxs.message[name] or mail.new_uuid()
```
Yeah, I know it is all mixed spaces and tabs but hoping to slowly move towards all tabs indentation.